WcaOnRails/app/mailers/competitions_mailer.rb

```diff
@@ -57,9 +57,10 @@ def submit_results_nag(competition)
 
   def submit_report_nag(competition)
     @competition = competition
+    delegates = competition.delegates.select(&:any_kind_of_delegate?)
     mail(
-      to: competition.delegates.pluck(:email),
-      cc: ["board@worldcubeassociation.org"] + competition.delegates.map { |delegate| delegate.senior_delegate&.email }.uniq.compact.flatten,
+      to: delegates.pluck(:email),
```

Comment:
What happens if this results in us emailing zero delegates? Does the mail stuff freak out (I have a vague recollection of it crashing if you pass an empty array here)?